src/background/index.ts

```
@@ -112,6 +114,27 @@ function initialize(): void {
+  // Listen for provider changes saved to chrome.storage.local (Chromium settings)
+  try {
+    chrome.storage.onChanged.addListener((changes: { [key: string]: chrome.storage.StorageChange }, areaName: string) => {
+      if (areaName !== 'local') return
+      const key = BROWSEROS_PREFERENCE_KEYS.PROVIDERS
+      const change = changes[key]
+      if (!change) return
+      try {
+        const raw = typeof change.newValue === 'string' ? JSON.parse(change.newValue) : change.newValue
+        const config = BrowserOSProvidersConfigSchema.parse(raw)
+        lastProvidersConfigJson = JSON.stringify(config)
+        try { langChainProvider.clearCache() } catch (_) {}
```

ESLint

[error] Empty block statement.

(no-empty)